server/controllers/admin.py

```diff
@@ -440,7 +467,7 @@ def templates(cid, aid):
         if files:
             templates = {}
             for template in files:
-                templates[template.filename] = str(template.read(), 'latin1')
+                templates[template.filename] = str(template.read(), 'utf-8')
```

Comment:
Any reason this was changed?  AFAIK `latin1` was used specifically instead of `utf-8`